www/artworks/new.php

```diff
@@ -116,6 +116,18 @@
 			</fieldset>
 			<fieldset id="pd-proof">
 				<legend>Proof of Public Domain Status</legend>
+				<p>PD proof must take the form of:</p>
+				<fieldset>
+					<label>
+						Link to <a href="/manual/latest/10-art-and-images#10.3.3.7.4">approved museum page</a>
```

Comment:
nit: Add "an" to the phrase to get "Link to an <a href="...">approved museum page</a>"